packages/cli/src/lib/errors/APIError.ts

```diff
@@ -1,22 +1,89 @@
-import {CLIBaseError} from './CLIBaseError';
+import {red} from 'chalk';
+import dedent from 'ts-dedent';
+import {PrintableError, SeverityLevel} from './printableError';
 
 export interface APIErrorResponse {
   message: string;
   errorCode: string;
   requestID: string;
 }
 
+export interface AxiosErrorFromAPI {
+  response: {
+    status: number;
+    data: APIErrorResponse;
+  };
```

Comment:
I took the opportunity to link the `APIErrorResponse` (returned when using the platform client) and `AxiosErrorFromAPI`.